x-pack/plugin/core/src/main/plugin-metadata/entitlement-policy.yaml

```diff
@@ -17,6 +17,8 @@ org.apache.httpcomponents.httpcore.nio:
 org.apache.httpcomponents.httpasyncclient:
   - manage_threads
 unboundid.ldapsdk:
+  - set_https_connection_properties
```

Comment:
How did you determine this one was necessary?

Reply:
Yeah the comment seems like a good idea, though I suppose anyone looking for the origin of that line will land on this PR and see this very conversation.